### PR TITLE
i52 math element wording

### PR DESCRIPTION
**Pull request by kerimoyle**
Addresses #52

### CODE REVIEW

**Comment by kerimoyle** [comment]
Rendered version here: https://cellml-specification-dev.readthedocs.io/en/i52_mathml_wording/reference/formal_and_informative/specB12.html

File: reference/sectionB_elements.inc [hunk 1]
```diff
@@ -399,21 +399,21 @@ Specific information items
...
 
-   3. Every variable name given using the MathML :code:`ci` element MUST be a :ref:`variable reference<specC_variable_reference>` to a :code:`variable` within the :code:`component` element that the :code:`math` element is contained.
+   3. Every variable name given using the MathML :code:`ci` element MUST be a :ref:`variable reference<specC_variable_reference>` to a :code:`variable` within the :code:`component` element in which the :code:`math` element is contained.
 
 .. marker_math_3
 
 .. container:: issue-math-cn-units-attribute
 
-   4. Any MathML :code:`cn` elements MUST each have an attribute in the :ref:`CellML namespace<specA_cellml_namespace>`, with a local name equal to :code:`units`. The value of this attribute MUST be a valid units reference.
+   4. A MathML :code:`cn` element MUST have an attribute in the :ref:`CellML namespace<specA_cellml_namespace>`, with a local name equal to :code:`units`. The value of this attribute MUST be a valid units reference to a :code:`units` item within the infoset or a :ref:`built-in units<table_built_in_units>` item.
```
Comment (MichaelClerx) [inline reference/sectionB_elements.inc]: "All MathML :code:`cn` elements MUST have an attribute"?

File: reference/sectionB_elements.inc [hunk 2]
```diff
@@ -441,7 +441,7 @@ Table: Supported MathML Elements
 +----------------------------------+----------------------------------+
 | Calculus Elements                | <diff>                           |
 +----------------------------------+----------------------------------+
-| Qualifier Elements               | <bvar>, <logbase>, <degree> *    |
+| Qualifier Elements               | <bvar>, <logbase>, <degree> \*   |
```
Comment (MichaelClerx) [inline reference/sectionB_elements.inc]: What's that star for?

Comment (kerimoyle) [reply reference/sectionB_elements.inc]: There's something under the table that was a note for this box?

Comment (kerimoyle) [reply reference/sectionB_elements.inc]: * (child of <root> or <diff>)

File: reference/sectionB_elements.inc [hunk 3]
```diff
@@ -399,21 +399,22 @@ Specific information items
...
-   3. Every variable name given using the MathML :code:`ci` element MUST be a :ref:`variable reference<specC_variable_reference>` to a :code:`variable` within the :code:`component` element that the :code:`math` element is contained.
+   3. Every variable name given using the MathML :code:`ci` element MUST be a :ref:`variable reference<specC_variable_reference>` to a :code:`variable` within the :code:`component` element in which the :code:`math` element is contained.
 
 .. marker_math_3
 
 .. container:: issue-math-cn-units-attribute
 
-   4. Any MathML :code:`cn` elements MUST each have an attribute in the :ref:`CellML namespace<specA_cellml_namespace>`, with a local name equal to :code:`units`. The value of this attribute MUST be a valid units reference.
+   4. A MathML :code:`cn` element MUST have an attribute in the :ref:`CellML namespace<specA_cellml_namespace>`, with a local name equal to :code:`units`. 
+   The value of this attribute MUST be a valid :ref:`units reference<specC_units_reference>` to a :code:`units` item within the infoset or a :ref:`built-in units<table_built_in_units>` item.
```
Comment (kerimoyle) [inline reference/sectionB_elements.inc]: remove end of sentence.

Comment (kerimoyle) [reply reference/sectionB_elements.inc]: Done in c389c83

**Pull request merged**